test_util/helpers.py

```diff
@@ -4,10 +4,34 @@
 import requests
 import retrying
 
+import test_util
+
 Host = namedtuple('Host', ['private_ip', 'public_ip'])
 SshInfo = namedtuple('SshInfo', ['user', 'home_dir'])
 
 
+class DcosUser:
+    """A lightweight user representation."""
+    def __init__(self, auth_json):
```

Comment:
nit: This feels a lot like a named tuple + a helper function, but I can also get doing a class.